transformation/transformationPlugins/jsonMessageTransformers/jsonTypeMappingsSanitizationTransformer/src/main/java/org/opensearch/migrations/transform/typemappings/SourceProperties.java

```diff
@@ -0,0 +1,21 @@
+package org.opensearch.migrations.transform.typemappings;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class SourceProperties {
+    private String type;
```

Comment:
Should the `type` be a String or an enumeration?  It feels like we want a constrained and obvious set of values for this one.